bin/phoenix_utils.py

```diff
@@ -71,6 +71,7 @@ def setPath():
     PHOENIX_CLIENT_JAR_PATTERN = "phoenix-*-client.jar"
     PHOENIX_THIN_CLIENT_JAR_PATTERN = "phoenix-*-thin-client.jar"
     PHOENIX_QUERYSERVER_JAR_PATTERN = "phoenix-*-queryserver.jar"
+    PHOENIX_LOADBALANCER_JAR_PATTERN = "phoenix-load-balancer-*[!t][!e][!s][!t][!s].jar"
```

Comment:
I think all of the extra square-brackets are unnecessary here. Wouldn't `phoenix-load-balancer-*[!tests].jar` be sufficient?

Reply:
Unfortunately, it is impossible. Here used fnmatch (https://docs.python.org/2/library/fnmatch.html) and [!tests] means matches any character not in "tests" (just 1 character in the end), so if we will have phoenix-load-balancer-someversion-snapshot.jar we will not find it.

Reply:
Bummer. Maybe we switch this over to just use normal Python `re` later. Fine for now -- thanks for clarifying!